go.mod

```diff
@@ -1,6 +1,8 @@
 module go.viam.com/utils
 
-go 1.21.13
+go 1.22
```

Comment:
This and the `toolchain` line are a byproduct of `go mod tidy`. Not sure if this is interesting